images/golang-tools/pipeline.config.json

```diff
@@ -2,12 +2,24 @@
   {
     "steps": {
       "build": {
-        "id": "build_docker_with_build_arg_arm_x86",
+        "id": "build_docker_with_build_arg",
         "env": {
           "BUILD_ARG": "GO_VERSION",
           "BUILD_ARG_VARIANTS": "1.8.7;1.9.4;1.10.3;1.10.4;1.11"
         }
       }
     }
+  },
+  {
+    "steps": {
+      "build": {
+        "id": "build_docker_with_build_arg_arm",
+        "env": {
+          "TARGET_ARCH": "arm32v7",
+          "BUILD_ARG": "GO_VERSION",
+          "BUILD_ARG_VARIANTS": "1.10;1.11"
```

Comment:
please use 1.10.4 instead of 1.10

Reply:
There is no tag for resin/armv7hf-golang:1.10.4 ;D Would suggest to go with this one till we proceeded with the multiarch idea

Reply:
Could offer you a 1.10.3 ;)

Reply:
I guess 1.10.3 is fine then

Reply:
or just leave the 1.10

Reply:
went with 1.10.2 (acutally used in client repo) and 1.10.3